### PR TITLE
EVEREST-633 Fix adding namespaces after initial install

### PR DESCRIPTION
**Pull request by recharte**
**Fix adding namespaces after initial install**
---
**Problem:**
EVEREST-633

With the initial implementation we weren't able to add new namespaces after initial install.

**Cause:**
The operator subscription was never update which lead to the DB_NAMESPACES env var not change so the everest-operator and everest BE wouldn't know about the new namespces.

**Solution:**
Update the subscription with a merged env var list.

**CHECKLIST**
---
**Jira**
- [x] Is the Jira ticket created and referenced properly?

**Tests**
- [ ] Is an Integration test/test case added for the new feature/change?
- [ ] Are unit tests added where appropriate?

### CODE REVIEW

**Review by michal-kralik — APPROVED** [review]

File: pkg/kubernetes/kubernetes.go [hunk 1]
```diff
@@ -629,47 +627,96 @@ type InstallOperatorRequest struct {
...
+					namespacesMap[ns] = struct{}{}
+				}
+				for _, ns := range cfgNamespaces {
+					namespacesMap[ns] = struct{}{}
+				}
+				namespaces := []string{}
+				for ns := range namespacesMap {
+					namespaces = append(namespaces, ns)
+				}
+				sub.Env[i].Value = strings.Join(namespaces, ",")
```
Comment (oksana-grishchenko) [inline pkg/kubernetes/kubernetes.go:664]: to improve readability perhaps it's worth it to move this part to a separate function that takes two strings and returns a string. It would also allow to add unit-tests for this function to make sure we covered all corner cases.

**Review by oksana-grishchenko — APPROVED** [review]
